### ISSUE TITLE
Missing script: "start"

### ISSUE THREAD

**Issue by Kakcalu13** [gazebo-web/gazebosim-app#13]
I was trying to load gzweb by `npm start` but it doesn't work for me.

The error output:
```
bwuk@robots:~/gzweb$ npm start
npm ERR! Missing script: "start"
npm ERR! 
npm ERR! Did you mean one of these?
npm ERR!     npm star # Mark your favorite packages
npm ERR!     npm stars # View packages marked as favorites
npm ERR! 
npm ERR! To see a list of scripts, run:
npm ERR!   npm run

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/bwuk/.npm/_logs/2022-11-20T05_00_08_399Z-debug-0.log
bwuk@robots:~/gzweb$ 

```

What I did was following steps:
```
nvm install 14
nvm use 14
node -v && npm -v # Your versions will be correct.
npm install
npm run test
npm run build
npm start
```

Right after npm start, it gives error like above. Is it expected?

**Comment by Kakcalu13** [comment]
I am able to connect websocket however I couldn't load js side. 

As for the gazebo side:
Here is what I did

1) Pick any world sdf. In my own case, it's `level_one.sdf` world so I ran 
`gz gazebo -v 4 -s -r level_one.sdf`
2) Open a new terminal and type `nano websocket.gzlaunch` then paste [this](https://raw.githubusercontent.com/gazebosim/gz-launch/gz-launch6/examples/websocket.gzlaunch) into this file.
3) `gz launch -v 4 websocket.gzlaunch`

I'm able to receive something from gazebo side but not doing what I wanted though. 

See here:
![image](https://user-images.githubusercontent.com/65916520/202886812-83c7a602-79cf-407e-9708-8afe9771a752.png)

I'm not sure how to set them both up.
